tests/helpers/unit-test-helpers.js

```
@@ -534,6 +534,332 @@ export function generateTestData(type, overrides = {}) {
+  // Create a default test transaction if not provided
+  let transactionId = overrides.transaction_id;
+  if (!transactionId) {
+    // Use createTestTransaction helper which has correct schema
+    const txn = await createTestTransaction(db, { event_id: overrides.event_id || 1 });
+    transactionId = txn.id;
+  }
+
+  const ticket = {
+    ticket_id: `TEST_${uniqueId}`,
+    event_id: 1,
+    transaction_id: transactionId,
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)